### DISCUSSION TITLE
Training a model on/for mkdocstrings

### DISCUSSION THREAD

**Discussion by rraulf** [mkdocstrings/mkdocstrings#684] [Ideas]
delete this discussion

**Comment by pawamoy** [comment]
There's a bit of data in our issues, discussions and docs :) A tiny bit more on StackOverflow. Maybe on Discord/Reddit too.

**Comment by rraulf** [comment]


**Reply by pawamoy** [reply]
This:

- https://mkdocstrings.github.io/python/usage/configuration/docstrings/
- https://mkdocstrings.github.io/griffe/guide/users/recommendations/docstrings/
- https://mkdocstrings.github.io/griffe/reference/docstrings/

**Reply by pawamoy** [reply]
Also there's more coming in mkdocstrings-python's docs: a proper guide with our recommendations on how to write docstrings that play best with mkdocstrings :) stay tuned 📻

**Comment by rraulf** [comment]
please delete this discussion

**Reply by pawamoy** [reply]
Why?
